diff -aurN models-2.3.0/research/slim/nets/superscaler_wrapper.py models/research/slim/nets/superscaler_wrapper.py
--- models-2.3.0/research/slim/nets/superscaler_wrapper.py	1970-01-01 08:00:00.000000000 +0800
+++ models/research/slim/nets/superscaler_wrapper.py	2020-10-28 21:48:45.732039337 +0800
@@ -0,0 +1,24 @@
+import tensorflow.compat.v1 as tf
+import tf_slim as slim
+from nets import inception
+
+
+def get_inceptionv4():
+    batch_size = 5
+    height, width = 299, 299
+    num_classes = 1000
+    inputs = tf.random.uniform((batch_size, height, width, 3))
+    labels = tf.zeros(shape=[batch_size], dtype=tf.int64, name="labels")
+    logits, _ = inception.inception_v4(inputs, num_classes)
+    cross_entropy = tf.nn.sparse_softmax_cross_entropy_with_logits(
+        labels=labels, logits=logits, name="cross_entropy_per_example")
+    loss = tf.reduce_mean(cross_entropy, name="cross_entropy")
+    opt = tf.train.GradientDescentOptimizer(0.123456)  # lr
+    grads = opt.compute_gradients(loss)
+    apply_gradient_op = opt.apply_gradients(grads)
+    return apply_gradient_op, loss
+
+
+def get_model(model_name):
+    models = {"inceptionv4": get_inceptionv4()}
+    return models[model_name]
